internal/core/window.rs

```diff
@@ -60,6 +60,11 @@ pub trait WindowAdapter {
     /// Returns the window API.
     fn window(&self) -> &Window;
 
+    /// Show the window if visible is true, hide otherwise.
```

Comment:
```suggestion
    /// Show the window if the argument is true, hide otherwise.
```

or

```suggestion
    /// Show the window if `_visible` is true, hide otherwise.
```